src/client.rs

```diff
@@ -114,6 +115,11 @@ impl Client {
         }
     }
 
+    /// Get the capabilities returned by the remote end when opening the session.
+    pub fn remote_capabilities(&self) -> &Capabilities {
```

Comment:
I think I'm okay with this just being called `capabilities` actually. I doubt anyone would expect that this is "the capabilities we originally sent to the server". And we can also clarify this in the doc text. What do you think?